roles/vdm/tasks/main.yaml

```diff
@@ -182,11 +183,35 @@
     - uninstall
     - update
 
+- name: Include Deployment Operator - Install
+  include_tasks: deployment_operator.yaml
+  when:
+    - V4_DEPLOYMENT_OPERATOR_ENABLED
+  tags:
+    - install
+    - update
+
+- name: Include Orchestration
```

Comment:
From the meeting, I decided to keep this as orchestration. In the README in sas-bases under examples it's referred to as the "sas-orchestration image" and in the online documentation it's "Orchestration Tool". So this name seems to be consistent with that.